snapshot.sh

```diff
@@ -11,11 +11,24 @@ SNAPSHOT_PREFIX="${SNAPSHOT_PREFIX:-$CHAIN_ID}"
 SNAPSHOT_RETAIN="${SNAPSHOT_RETAIN:-2 days}"
 SNAPSHOT_METADATA="${SNAPSHOT_METADATA:-1}"
 SNAPSHOT_SAVE_FORMAT="${SNAPSHOT_SAVE_FORMAT:-$SNAPSHOT_FORMAT}"
-valid_snapshot_formats=(tar tar.gz)
+valid_snapshot_formats=(tar tar.gz tar.zst)
 # If not one of valid format values, set it to default value
-if ! echo ${valid_snapshot_formats[@]} | grep -qiw -- "$SNAPSHOT_SAVE_FORMAT"; then
+if ! echo "${valid_snapshot_formats[@]}" | grep -qiw -- "$SNAPSHOT_SAVE_FORMAT"; then
   SNAPSHOT_SAVE_FORMAT=tar.gz
 fi
+# Actual valid values not documented
```

Comment:
Have never used the `--long` .. I don't think many know / will use it.
How about leaving it out until it is clear what is it for, what are the best options and it seems that `--memory` needs to be passed to the decompressor based on this note:

> Note: If windowLog is set to larger than 27, --long=windowLog or --memory=windowSize needs to be passed to the decompressor.
> https://manpages.debian.org/unstable/zstd/zstd.1.en.html